helm/charts/determined/templates/db-volume-snapshot-class.yaml

```diff
@@ -1,6 +1,9 @@
+{{- if and .Release.IsUpgrade .Values.db.snapshotSuffix }}
 apiVersion: snapshot.storage.k8s.io/v1
 kind: VolumeSnapshotClass
 metadata:
   name: determined-db-snapshot-class
 driver: pd.csi.storage.gke.io
 deletionPolicy: Retain
+{{ end }}
```

Comment:
```suggestion
{{- end }}
```

Reply:
this is just a guess. i think without this you get an extra newline, but helm is weird.

Reply:
is this a syntax preference? we're not consistent about this in other helm preferences

Reply:
the `-` just means drop all whitespace to the left. i'm not sure if it will drop the newline for an empty line or not. but i thought with how this is you may end up with 2 newlines when you run `helm template`.

Reply:
doesn't seem to make a difference because it looks like helm trims extra whitespace at the end of files by default.

Reply:
if you throw a comment at the end of the file it'll preserve the extra newline and you can see what i was pointing out then.

Reply:
anyway, haha, lgtm. just a digression on helm.